core/test/nuxeo-task-page-provider.test.js

```diff
@@ -0,0 +1,204 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+import '../nuxeo-task-page-provider.js';
+import fakeServer from './nuxeo-mock-client';
+import { fixture, html, waitForEvent } from './test-helpers';
+
+/* eslint-disable no-unused-expressions */
+suite('<nuxeo-task-page-provider>', () => {
+  suite('REST Calls', () => {
+    let server;
+    setup(() => {
+      server = fakeServer.create();
+    });
+
+    teardown(() => {
+      server.restore();
+    });
+
+    suite('Responses and Events', () => {
+      let pageProvider;
+      setup(async () => {
+        pageProvider = await fixture(html`
+          <nuxeo-task-page-provider></nuxeo-task-page-provider>
+        `);
+
+        /*
+         * This workaround is needed due to the 'error' event fired in component.
+         * More Details: https://github.com/webcomponents/webcomponentsjs/issues/138
+         */
+        pageProvider.addEventListener('error', (event) => {
+          event.stopPropagation(); // avoid browser to interpret it.
+        });
+      });
+
+      test('Should fire "error" event when server returns fetch error', async () => {
+        server.rejectWith('get', '/api/v1/task');
+
```

Comment:
this test is the one where there is no server response, is it right ?

Reply:
Not really, there is a server response, an error.

Reply:
yep, i asked because i tried to compare the `'Should throw an error when server returns fetch error'` the tests just below this one